Source/Evolve/WarpXEvolveEM.cpp

```diff
@@ -92,13 +92,13 @@ WarpX::EvolveEM (int numsteps)
             FillBoundaryB(guard_cells.ng_alloc_EB, guard_cells.ng_Extra);
             UpdateAuxilaryData();
             // on first step, push p by -0.5*dt
-            for (int lev = 0; lev <= finest_level; ++lev) {
+            for (int lev = 0; lev <= finest_level; ++lev)
+            {
```

Comment:
In general, could you avoid these style changes in a non-style-related PR? They dilute a little bit the topic of this PR (no need to remove them from this PR as there are not too many of these changes). Thanks!

Reply:
Yes, will avoid this in the future!